### ISSUE TITLE
Leading Dijet invariant mass

### ISSUE THREAD

**Issue by jslawless** [lawrenceleejr/FragmentationStudy#26]
![Screenshot 2024-04-25 at 5 56 34 PM](https://github.com/lawrenceleejr/FragmentationStudy/assets/116592104/ffb6ceb0-8c0d-4d1e-9729-0f01637378f9)

Run on the 1997 file. Does this make sense?

**Comment by lawrenceleejr** [comment]
Unclear! Can you describe the selection in full? Is this with the photon requirement?

**Comment by jslawless** [comment]
No, I put no requirements on this. Just plotted it straight from that ak4 tree

**Comment by lawrenceleejr** [comment]
Also useful if you can make the distribution of "Energy" for the event selection you have. This to me looks like it's dominated by Z peak runs and is somehow something like the leading two jets in trijet events in Z peak runs.

**Comment by lawrenceleejr** [comment]
I see. Then it's kind of impossible to understand. Would need more control of the collision energy and njet to understand.

**Comment by jslawless** [comment]
Can I assume that the events in the different trees are in the same order?

**Comment by lawrenceleejr** [comment]
Likely but you should be able to do a TTree Scan of the Event numbers to see.

**Comment by jslawless** [comment]
That run was exactly at the Z peak I think. I don't think the ak4 branch has event numbers.
